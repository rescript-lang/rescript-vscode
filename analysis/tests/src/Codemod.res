type someTyp = [#valid | #invalid]

let ff = (v1: someTyp, v2: someTyp) => {
  let x = switch (v1, v2) {
  //      ^c-a (#valid, #valid) | (#invalid, _)
  | (#valid, #invalid) => ()
  }
  x
}
