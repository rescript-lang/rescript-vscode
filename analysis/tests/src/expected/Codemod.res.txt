Codemod AddMissingCasessrc/Codemod.res 3:10
switch (v1, v2) {
          //      ^c-a (#valid, #valid) | (#invalid, _)
          | (#valid, #invalid) => ()
          | (#valid, #valid) => failwith("TODO")
          | (#invalid, _) => failwith("TODO")
          }

